third_party/java/arcs/build_defs/internal/tools.oss.bzl

```diff
@@ -0,0 +1,23 @@
+"""Rules that invoke sigh scripts."""
+
+load("//third_party/java/arcs/build_defs:sigh.bzl", "sigh_command")
+
+def arcs_manifest_parse_test(name, srcs, deps = []):
```

Comment:
Add visibility

Reply:
hmm, if this isn't being called direction, then it's fine if we don't include visibility.

Reply:
I'm not sure what you mean by "direction".

But this is a test, it doesn't have any useful output. No other rules will depend on it, so there's no need for visibility.